src/pages/events/commands.md

```diff
@@ -84,24 +84,30 @@ where:
 
 You can subscribe to any available observer event. You cannot subscribe to a plugin event unless it was registered in the `app/etc/config.php` file and subsequently unsubscribed with the [`events:unsubscribe` command](#unsubscribe-from-a-commerce-event). [Register events](./module-development.md#register-events describes the format of these files.)
 
+Adobe Commerce does not send all event fields to your external application by default. Instead, you must use the `--field` option to specify which fields to transmit. To send all event fields, must specify a separate `--field` option for each field. This practice keeps data transmission to a minimum and helps ensure sensitive information is not sent by mistake. If the Commerce event contains objects, use dotted notation to specify fields within an object. For example, For example, if your event contains a `stock_data` object, and you want to send its `product_id` and `qty` fields, you would specify the `--fields stock_data.product_id` and `--fields stock_data.qty` command options. [Commerce module development](./module-development.md) provides a detailed example using files to register events.
+
 ### Usage
 
-`bin/magento events:subscribe <event_code>`
+`bin/magento events:subscribe <event_code> --fields=<name1> --fields=<name2>`
 
 ### Arguments
 
 `<event_code>` Required. Specifies the event to subscribe to. The value must in the format `<type.event_name>`.
 
+### Options
+
+`--fields <field_name>` Required. An event field to transmit to the Adobe App Builder application. You can specify this option multiple times. Each instance can contain only one field name.
+
 ### Example
 
 ```bash
-bin/magento events:subscribe observer.customer_login
+bin/magento events:subscribe com.adobe.commerce.observer.catalog_product_save_after --fields sku --fields stock_data.qty 
```

Comment:
`com.adobe.commerce` prefix is added automatically and must not be presented in the `event:subscribe` command. The command will return an error as it expects that event_code will start with **plugin** or **observer**.